ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/result-add/index.tsx

```diff
@@ -0,0 +1,104 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import { hot } from 'react-hot-loader'
+import styled from '../../styles/styled-components'
+import MarionetteRegionContainer from '../../container/marionette-region-container'
+import * as PopoutView from '../../../component/dropdown/popout/dropdown.popout.view'
+import * as ListCreateView from '../../../component/list-create/list-create.view'
+
+type Result = {
```

Comment:
could you export this type from the container component and import it here?

Reply:
It's not used in the container. I think it makes sense to leave it here unless we see a need to reuse it.